Packs/FiltersAndTransformers/Scripts/IfElif/IfElif.py

```diff
@@ -0,0 +1,76 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import re
+
+ARGS = demisto.args()
+CONTEXT = demisto.context()
+OVERRIDE_BUILTINS = {'__builtins__': None}
+CONTEXT_REGEX = re.compile('${([\S]+?)}')
+
+
+def get_from_context(path: str) -> Any:
+    '''Gets a value of the format from the context.'''
+
+    result = dict_safe_get(CONTEXT, path.split('.'), KeyError)
+
+    if result is KeyError:
+        raise KeyError(f'Missing key {path!r} in context.')
+
+    return result
+
+
+def parse_condition_for_context_keys(condition: str) -> str:
+    '''Parses a string for context keys.'''
+
+    for match in CONTEXT_REGEX.findall(condition):
+        replacement = get_from_context(match)
+        condition = condition.replace(f'${{{match}}}', json.dumps(replacement), 1)
+
+    return condition
+
+
+def evaluate_condition(condition: str) -> bool:
+
+    try:
+        parsed_condition = parse_condition_for_context_keys(condition)
+        result = eval(parsed_condition, OVERRIDE_BUILTINS)  # noqa: PGH001
```

Comment:
isn't `ast.literal_eval()` safer and enough for us?

Reply:
`ast.literal_eval` doesn't parse expressions